Upstream: No
Reason: Without this patch some tests fail as the server is not fully started,
issue only present on Alpine running in docker
--- a/t/Util.pm
+++ b/t/Util.pm
@@ -125,7 +125,7 @@
                 if (waitpid($pid, WNOHANG) == $pid) {
                     die "server failed to start (got $?)\n";
                 }
-                sleep 0.1;
+                sleep 3;
             }
         }
         my $guard = scope_guard(sub {
